src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java

```diff
@@ -77,6 +77,17 @@ public AboutDialogContents(ProductInfo info, ProductEditionInfo editionInfo)
       productName.setText(editionInfo.editionName());
       copyrightYearLabel.setText("2009-" + info.copyright_year);
 
+      // Warn that dailies and previews aren't supported
+      if (info.build_type.equals("Daily") || info.build_type.equals("Preview")) {
```

Comment:
I wonder if this should check `build_type != "Release"`? Assuming that will be the only "official" label used for releases.